config.js

```diff
@@ -1,7 +1,9 @@
-const PORT = process.env.PORT || (process.env.NODE_ENV == "test" ? 9444 : 9443);
+const PORT = process.env.PORT || 9444;
+const BASE_URL = 'bulk-data-server';
+const SERVER_BASE_URL = process.env.BASE_URL
 module.exports = {
 
-    baseUrl: process.env.BASE_URL || `http://localhost:${PORT}`,
+    baseUrl: SERVER_BASE_URL ? `${SERVER_BASE_URL}/bulk-data-server` : `http://localhost:${PORT}/bulk-data-server`,
```

Comment:
should we be hardcoding the string `bulk-data-server`?

Reply:
fair point, let me see if I can put this value in a variable. it might cause issues with some of the import paths in the html files but I may be able to use templating to handle that

Reply:
fixed for now, but note that the html pages still have /bulk-data-server/ hardcoded into their path, I may look into changing this in a future issue.